synapse/config/repository.py

```diff
@@ -202,6 +215,21 @@ def generate_config_section(self, data_dir_path, **kwargs):
 
         return (
             r"""
+        ##
+        ## Media Store
+        ##
+
+        # Is an external media repo in use?
+        #
+        # Set 'False' to use Synapse's media repo. If this is set to 'True',
+        # Synapse's media repo below must be disabled.
+        #
+        #external_media_repo: False
```

Comment:
this should be ready for uncommenting:

```suggestion
        #external_media_repo: True
```